can_tools/scrapers/official/AZ/AZ_TableauPull.py

```diff
@@ -0,0 +1,280 @@
+                dataUrl = f'https://tableau.azdhs.gov{tdata["vizql_root"]}/bootstrapSession/sessions/{tdata["sessionid"]}'
+                r = requests.post(
+                    dataUrl,
+                    data={
+                        "sheet_id": tdata["sheetId"],
+                        "showParams": tdata["showParams"],
+                    },
+                )
+                # Regex the non-json output
+                dat = re.search("\d+;({.*})\d+;({.*})", r.text, re.MULTILINE)
+                # load info head and data group separately
+                info = json.loads(dat.group(1))
+                fdat = json.loads(dat.group(2))
+                # Get county data
+                cy = extractData(fdat, county[1])
+                outDf = outDf.append(cy, ignore_index=True)
+
+        outDf["CumPosTests"] = outDf["PercentPositive"] * outDf["Number of tests"]
+        outDf["CumDiagPosTests"] = (
+            outDf["Percent Positive Diagnostic tests"]
+            * outDf["Number of Diagnostic tests"]
+        )
+
+        # NOTE: There is currently a bug in the AZDHS dashboard summary page. They do NOT show correct values for antibody positivity rate
+        outDf["CumSeroPosTests"] = outDf["CumPosTests"] - outDf["CumDiagPosTests"]
+        if self.location_type == "county":
+            outDf = outDf[outDf["location"] != 4]
+        return outDf
+
+    def normalize(self, data):
+        df = data
+        crename = {
+            "New Cases": CMU(category="cases", measurement="new", unit="people"),
+            "New Deaths": CMU(category="deaths", measurement="new", unit="people"),
+            "New Tested": CMU(
+                category="unspecified_tests_total", measurement="new", unit="specimens"
+            ),
+            "New Tested PCR": CMU(
+                category="antigen_pcr_tests_total", measurement="new", unit="specimens"
+            ),
+            "New Tested serology": CMU(
+                category="antibody_tests_total", measurement="new", unit="specimens"
+            ),
+            "Number of Cases": CMU(
+                category="cases", measurement="cumulative", unit="people"
+            ),
+            "Number of Diagnostic tests": CMU(
+                category="antigen_pcr_tests_total",
+                measurement="cumulative",
+                unit="specimens",
+            ),
+            "Number of deaths": CMU(
+                category="deaths", measurement="cumulative", unit="people"
+            ),
+            "Number of tests": CMU(
+                category="unspecified_tests_total",
+                measurement="cumulative",
+                unit="specimens",
+            ),
+            "Number of tests serology": CMU(
+                category="antibody_tests_total",
+                measurement="cumulative",
+                unit="specimens",
+            ),
+            "CumPosTests": CMU(
```

Comment:
Is the unspecified due to the fact that `CumPosTests` could be (and probably is) both diagnostics and serology?

Reply:
This is correct - the value is a derivation from "Percent positive" value, which includes PCR, antigen, serology together.